hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieParquetInputFormat.java

```diff
@@ -253,4 +253,4 @@ private BootstrapBaseFileSplit makeExternalFileSplit(PathWithBootstrapFileStatus
       throw new HoodieIOException(e.getMessage(), e);
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Can we revert this change ? Please avoid formatting changes to creep into the PR

Reply:
Done